tests/WE2E/test_configs/data_assimilation/config.process_obs.yaml

```diff
@@ -0,0 +1,14 @@
+metadata:
+  description: |-
+    Tests the data retrieval and data preprocessing tasks that are
+    precursors to running any data assimialtion components. Datasets
+    include bufr, lightning, and radar reflectivity.
+
+workflow:
+  PREDEF_GRID_NAME: RRFS_CONUS_25km
+  DATE_FIRST_CYCL: '2019061500'
+  DATE_LAST_CYCL: '2019061500'
```

Comment:
I'll need some feedback from @EdwardSnyder-NOAA on what data has been staged here before this information can be nailed down.

Reply:
I placed the predefined grid data on Hera, Jet, and Orion for the following grids: `RRFS_NA_3km`, `RRFS_NA_13km`, and `RRFS_CONUS_3km`. The test lightning data is stored on Jet and Orion for the 2022-07-20 00Z case. We still need radar and buffer obs data for that date. Before I add those obs data, I just want to make sure that this is the case date we want to go with. My thinking was that this case could be the sample case we provide to the community for the release.

Reply:
@EdwardSnyder-NOAA I'm sorry for the delay on this one.

I think we should pose this question at a release meeting. More generally "is there an interesting case(s) that we should use for the release?"